bindata/etcd/pod.yaml

```diff
@@ -151,6 +151,8 @@ ${COMPUTED_ENV_VARS}
         env | grep ETCD | grep -v NODE
 
         set -x
+        time etcdctl defrag
```

Comment:
you will need to set --command-timeout not 100% sure what would be global but I think 45s should be reasonable.

Reply:
we might need to retry in case of failure so for example if we timeout we need to retry until success.

Reply:
also as we want to make sure we are defraging the store explicitly  which needs `-data-dir=`

Reply:
actually disregard `--command-timeout` as we are not on the wire.